products/compute/autoscaler_properties.yaml

```diff
@@ -136,6 +136,42 @@
             - :DELTA_PER_SECOND
             - :DELTA_PER_MINUTE
           required: true
+        - !ruby/object:Api::Type::String
+          name: 'filter'
+          description: |
+            A filter string to be used as the filter string for
+            a Stackdriver Monitoring TimeSeries.list API call.
+            This filter is used to select a specific TimeSeries for
+            the purpose of autoscaling and to determine whether the metric
+            is exporting per-instance or per-group data.
+
```

Comment:
FYI, this won't actually show up as paragraphs in our docs: https://github.com/terraform-providers/terraform-provider-google-beta/blob/a1327d0404536886b2eea8aed52ab5f269867ded/website/docs/r/compute_autoscaler.html.markdown

I don't think you should make any changes, just more letting you know so you're not surprised when the docs have a big paragraph.